### ISSUE TITLE
Issues for iOS 11.0.x users

### ISSUE THREAD

**Issue by httpwemel** [ioscreatix/LittleX#24]
- Lock Screen does not have the swipe up bar, flashlight and camera buttons
- Unlocking the phone with a passcode only by swiping up does not automatically unlock the device (this is is not an issue when the home button is pressed)
- Button combinations with the volume buttons dont work (Screenshots and power off menu)
- Control center, whilst swiping up from the right edge works, it's the pre-iPhone X look
- turning on swipe to kill puts the device on safe mode
- Dock would sometimes continue to appear in multitasking (though just for quite a bit)

Still a very nice and customizable tweak :+1:  (can post pics if needed)

**Comment by cuza** [comment]
Yup, iPhone 6, iOS 11.0.3 and got all those issues mentioned above. 

PS: It's an awesome tweak hope it get fixed soon

**Comment by zyan910** [comment]
Afaik, It's because the iPX files aren't available in iOS 11.0.3 and below. Someone needs to port this.
